### PR TITLE
Update more Availability attributes in files not processed

### PR DESCRIPTION
**Pull request by chamons**
This touches some additional files https://github.com/xamarin/xamarin-macios/pull/3096 is having trouble with.

### CODE REVIEW

File: src/UIKit/UIStringDrawing.cs [hunk 1]
```diff
@@ -9,84 +9,96 @@
 
 namespace XamCore.UIKit {
 	public unsafe static partial class UIStringDrawing  {
-		[Availability (Introduced = Platform.iOS_2_0, Deprecated = Platform.iOS_7_0, Message = "Use NSString.DrawString(CGPoint, UIStringAttributes) instead.")]
+		[iOS (2, 0)]
```
Comment (spouliot) [inline src/UIKit/UIStringDrawing.cs]: anything < 6.0 can be dropped

File: src/AudioUnit/AudioUnit.cs [hunk 2]
```diff
@@ -1382,11 +1382,14 @@ enum AudioUnitPropertyIDType { // UInt32 AudioUnitPropertyID
 		ReverbRoomType = 10,
 		UsesInternalReverb = 1005,
 		SpatializationAlgorithm = 3000,
-		[Availability (Introduced = Platform.iOS_3_0, Deprecated = Platform.iOS_9_0)]
+		[iOS (3, 0)]
```
Comment (spouliot) [inline src/AudioUnit/AudioUnit.cs]: anything < 6.0 can be dropped

File: src/AudioUnit/AudioUnit.cs [hunk 3]
```diff
@@ -1654,10 +1657,11 @@ public enum SpatialMixerAttenuation {
 	}
 
 	[Flags]
-	[Availability (Introduced = Platform.iOS_8_0)]
+	[iOS (8, 0)]
 	public enum SpatialMixerRenderingFlags {
 		InterAuralDelay = (1 << 0),
-		[Availability (Introduced = Platform.iOS_3_0, Deprecated = Platform.iOS_9_0)]
+		[iOS (3, 0)]
```
Comment (spouliot) [inline src/AudioUnit/AudioUnit.cs]: < 6.0

File: src/GameKit/GameKit.cs [hunk 4]
```diff
@@ -24,14 +24,16 @@ namespace XamCore.GameKit {
 #endif
 	[NoWatch]
 	[Native]
-	[Availability (Introduced = Platform.iOS_3_0, Deprecated = Platform.iOS_7_0)]
+	[iOS (3, 0)]
```
Comment (spouliot) [inline src/GameKit/GameKit.cs]: < 6.0

File: src/AppKit/Enums.cs [hunk 5]
```diff
@@ -138,9 +139,11 @@ public enum NSComposite : nuint_compat_int {
 
 	[Native]
 	public enum NSBackingStore : nuint_compat_int {
-		[Availability (Introduced = Platform.Mac_10_0, Deprecated = Platform.Mac_10_13, Message = "Use 'Buffered' instead.")]
+		[Mac (10, 0)]
```
Comment (timrisi) [inline src/AppKit/Enums.cs]: Similar to the iOS < 6.0, we should be able to drop Mac tags < 7.0

Comment (VincentDondain) [reply src/AppKit/Enums.cs]: Mac tags < 10.7 (:

**Comment by chamons** [comment]
I agree we should do a scrub of "too old" attributes, but I don't feel like now is the best time (let's land PMCS changes while things are quiet).

**Pull request merged**